examples/vision/oml_omniglot.py

```diff
@@ -37,15 +37,25 @@ def fast_adapt(batch, learner, loss, adaptation_steps, shots, ways, device):
     for step in range(adaptation_steps):
         train_error = loss(learner(adaptation_data), adaptation_labels)
         train_error /= len(adaptation_data)
-        learner.adapt(train_error)
+        learner.adapt(train_error, freeze_feature_extraction=freeze_feature_extraction)
 
     # Evaluate the adapted model
     predictions = learner(evaluation_data)
     valid_error = loss(predictions, evaluation_labels)
     valid_error /= len(evaluation_data)
     valid_accuracy = accuracy(predictions, evaluation_labels)
-    return valid_error, valid_accuracy
+    return valid_error, valid_accuracy, evaluation_data, evaluation_labels
 
+def sample_random_set(batch, shots, ways, device):
```

Comment:
Random set should sample from all tasks? How much samples exactly are sampled in OML?